pom.xml

```diff
@@ -117,11 +118,13 @@
         <groupId>com.google.code.findbugs</groupId>
         <artifactId>jsr305</artifactId>
         <version>2.0.1</version>
+        <scope>compile</scope>
       </dependency>
       <dependency>
         <groupId>com.google.errorprone</groupId>
         <artifactId>error_prone_annotations</artifactId>
         <version>2.0.8</version>
+        <scope>compile</scope>
```

Comment:
iirc, it was a try of copying only the needed dependencies. See https://github.com/google/google-java-format/pull/52#issuecomment-226448224 ... i'll remove it, if it was/is no longer necessary.